api/golang/core/lib/enclaves/enclave_context.go

```diff
@@ -84,9 +85,10 @@ func (enclaveCtx *EnclaveContext) RunStarlarkScript(
 	dryRun bool,
 	parallelism int32,
 	experimentalFeatures []kurtosis_core_rpc_api_bindings.KurtosisFeatureFlag,
+	connect kurtosis_core_rpc_api_bindings.Connect,
```

Comment:
this ( and the relevant package call) would be a breaking change to the SDK we should rename the PR as such to reflect

it seems like we only change the non blocking endpoints and not the blocking SDK calls, perhaps as we only want to change the CLI. Should we be consistent?

Reply:
I updated the non blocking calls.